docker-compose-services/minio/README.md

```diff
@@ -0,0 +1,63 @@
+# MinIO
+
+This recipe adds a [MinIO](https://min.io/) container to a project.
+
+[MinIO](https://min.io/) is a S3 compatible object-storage.
+It can be used, to run an S3 instance locally, in case the hosted S3 is not reachable (like developing while offline)
+
+## Installation
+
+* Copy the `docker-compose.minio.yaml` file to your project's DDEV root folder (`.ddev/docker-compose.minio.yaml`)
+* Copy the `minio-build` folder to your project's DDEV root folder (`.ddev/minio-build`)
+* Copy the `commands/minio` directory to the commands folder of your project's DDEV commands folder (`.ddev/commands/minio/mc`) and make sure it's executable, `chmod +x .ddev/commands/minio/mc`.
+* Start (or restart) DDEV to have the service initialized: `ddev start`
+
+## Connection
+* MinIO is available at `ddev-<projectname>-minio:9000` **inside the containers** and at `https://<projectname>.ddev.site:9000` **outside the containers**
+* If `https://<projectname>.ddev.site:9000` is opened in the browser, it will redirect to the object browser
+
+## Configuration
+The server is configured through the environment variables in `docker-compose.minio.yaml`
+Default credentials (for API and  object browser):
+* MINIO_ACCESS_KEY=minio
+* MINIO_SECRET_KEY=minio123
```

Comment:
I wonder if these would be better to put in config.yaml these days, rather than having people edit the docker-compose.minio.yaml.